docs/en_US/ch1/exchangis_deploy_en.md

```diff
@@ -273,12 +296,12 @@ Please visit the Exchange front-end page at  http://${EXCHANGIS_INSTALL_IP}:8098
 
 ## 3. DSS ExchangisAppConn installation and deployment 
 
-If you want to use Exchangis1.0.0 front-end, you also need to install the DSS ExchangisAppConn plugin. Please refer to: [ExchangisAppConn installation documentation for plugins ](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0/docs/en_US/ch1/exchangis_appconn_deploy_en.md)
+If you want to use Exchangis1.0.0 front-end, you also need to install the DSS ExchangisAppConn plugin. Please refer to: [ExchangisAppConn installation documentation for plugins ](./docs/en_US/ch1/exchangis_appconn_deploy_en.md)
 
 ## 4. Linkis Sqoop engine installation and deployment 
 
-If you want to execute the Sqoop operation of Exchangis1.0.0 normally, you also need to install the Linkis Sqoop engine. Please refer to: : [Linkis Sqoop engine installation documentation ](https://linkis.apache.org/zh-CN/docs/latest/engine-usage/sqoop)
+If you want to execute the Sqoop operation of Exchangis1.0.0 normally, you also need to install the Linkis Sqoop engine. Please refer to: : [Linkis Sqoop engine installation documentation ](https://linkis.apache.org/zh-CN/docs/1.1.2/engine-usage/sqoop/)
```

Comment:
not the latest linkis version?

Reply:
It has been updated with the latest link.